lib/hoverMain.js

```diff
@@ -15,14 +21,17 @@
     activate (state) {
       this.subscriptions = new CompositeDisposable();
     },
-    this.subscriptions.add(atom.texteditor.getWordUnderCursor) {
+    this.subscriptions.add(atom.texteditor.getWordUnderCursor()) {
+      //add something here but unsure what that should be.
+      //Maybe add the console log for this. 
```

Comment:
Yes - this looks like the hook function we want. Go ahead and try some logging here.

Reply:
Note: you will need this to get hooked by making sure the Atom plugin loader executes this as an `onload`

Reply:
> Note: you will need this to get hooked by making sure the Atom plugin loader executes this as an `onload`

Also shouldn't this be handled by the state activation to force itself into activation?